core/src/cost_update_service.rs

```diff
@@ -157,8 +157,12 @@ impl CostUpdateService {
     ) -> u64 {
         let mut update_count = 0_u64;
         for (program_id, program_timings) in &mut execute_timings.details.per_program_timings {
-            let current_estimated_program_cost =
-                cost_model.read().unwrap().find_instruction_cost(program_id);
+            let current_estimated_program_cost = {
+                match cost_model.read().unwrap().find_instruction_cost_raw(program_id) {
```

Comment:
nit: Can we rename this to `find_average_instruction_cost`, and rename the current `find_instruction_cost` to `find_inflated_instruction_cost()`